metrics.proto

```diff
@@ -15,6 +15,8 @@ message Event {
         SessionEventPayload session_event_payload = 5;
         SessionStatisticsPayload session_statistics_payload = 6;
         ProposalPayload proposal_payload = 7;
+        SessionConnectError session_connect_error_payload = 8;
+        SessionConnectSuccess session_connect_success_payload = 9;
```

Comment:
`SessionConnectSuccess` and `SessionConnectError` look very similar and probably mutually exclusive.
Maybe we can have only single `SessionConnectionStatus`?